web/quickfeed_service.go

```diff
@@ -84,7 +82,7 @@ func (s *QuickFeedService) UpdateUser(ctx context.Context, in *connect.Request[q
 	}
 	if _, err = s.updateUser(usr, in.Msg); err != nil {
 		s.logger.Errorf("UpdateUser failed to update user %d: %v", in.Msg.GetID(), err)
-		return nil, status.Error(codes.InvalidArgument, "failed to update user")
+		return nil, connect.NewError(connect.CodeInvalidArgument, errors.New("failed to update user"))
```

Comment:
Should this be `InvalidArgument` instead?

Reply:
Not sure if I understand. What are alternatives?

Reply:
Hmm. I thought this was a failed precondition error. Maybe I made a mistake.